### PR TITLE
Outdated links in the about page #4771

### LINKED ISSUE #4771: Outdated links in the about page
In the about page (again),

there are links tied to contributors when TEAMMATES was hosted in Google code.

screen shot 2016-02-15 at 12 26 44

However, all links to their contributions now leads to this same <a href = "https://code.google.com/archive/p/teammatespes/"> page</a>.

screen shot 2016-02-15 at 12 27 04

Is it better to list their names only since the links does not provide any additional information?

### PR DESCRIPTION
**Pull request by yvsssantosh**
Fixes #4771

### CODE REVIEW

**Comment by kanghj** [comment]
@yvsssantosh, instead of starting a new pull request, actually you can fix your merge conflicts and continue on the other branch

**Comment by yvsssantosh** [comment]
Hi,
I accidentally deleted my repo so I couldnt fix my merge conflicts
I apologize for the inconvenience caused

File: src/main/webapp/about.html [hunk 1]
```diff
@@ -638,142 +544,92 @@ <h1 id="caption">About Us</h1>
...
-                        <li>Prithviraj Billa 
-                            [<a href="https://code.google.com/p/teammatespes/issues/list?can=1&q=prithviraj.m.billa"
-                                target="_blank">bug reporting/fixing, enhancements</a>]
+                        <li>Prithviraj Billa [bug reporting/fixing, enhancements]
                         </li>
                         <li>Rajat Saxena
                             [<a href="https://github.com/TEAMMATES/teammates/issues?q=involves%3Asrajat"
                                 target="_blank">bug reporting/fixing, enhancements</a>]
                         </li>
-                        <li>Shiv Kandikuppa
```
Comment (thyageshm) [inline src/main/webapp/about.html]: Why are these changes in this PR?

File: src/main/webapp/about.html [hunk 2]
```diff
@@ -638,142 +544,92 @@ <h1 id="caption">About Us</h1>
...
-                            [<a href="https://code.google.com/p/teammatespes/issues/list?can=1&q=chanjunweimy"
-                                target="_blank">bug reporting/fixing, enhancements</a>]
+                                target="_blank">bug fixing</a>]
                         </li>
-                        <li>Justin Ouyang 
-                            [<a href="https://code.google.com/p/teammatespes/issues/list?can=1&q=justinouyang2"
-                                target="_blank">bug reporting/fixing, enhancements</a>]
+                        <li>Junwei Chan [bug reporting/fixing, enhancements]
                         </li>
-                        <li>Leonard Hio
```
Comment (thyageshm) [inline src/main/webapp/about.html]: Also, this name has been completely removed!

**Comment by thyageshm** [comment]
Hey could we follow the existing style by having the `[...]` in the next line (and indented)?
Also, there are some names completely removed, and others shifted places...

File: src/main/webapp/about.html [hunk 3]
```diff
@@ -321,24 +321,18 @@ <h1 id="caption">About Us</h1>
...
-                                target="_blank">bug reporting/fixing, enhancements</a>]
+                        <li>Ang Ji Kai [bug reporting/fixing, enhancements]
                         </li>
                         <li>Ang Jin Juan Gabriel 
                             [initial adoption of Selenium]
                         </li>
-                        <li>Ang Nickolas 
-                            [<a href="https://code.google.com/p/teammatespes/issues/list?can=1&q=angnickolas"
-                                target="_blank">bug reporting/fixing, enhancements</a>]
+                        <li>Ang Nickolas
```
Comment (thyageshm) [inline src/main/webapp/about.html]: This person has no `[...]`

**Comment by yvsssantosh** [comment]
Sure....I'm working on it
Thank you for the review

**Comment by yvsssantosh** [comment]
@thyageshm 

Hi,
I've made changed to the code

Kindly Review :)

**Pull request merged**